### DISCUSSION TITLE
INTERNAL_SERVER_ERROR Forbidden while creating page from GraphQL

### DISCUSSION THREAD

**Discussion by gmouawad** [requarks/wiki#5601] [Help / Questions] [answered]
**I am trying to use GraphQL to create a page, I have added an http header with the authorization bearer token. When I execute the mutation to create a wiki js page, I am getting the below error (I hope you can help me):**

{
  "errors": [
    {
      "message": "Forbidden",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "pages",
        "create"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "Error: Forbidden",
            "    at field.resolve (/wiki/server/graph/directives/auth.js:47:17)",
            "    at field.resolve (/wiki/node_modules/apollo-server-core/dist/utils/schemaInstrumentation.js:52:26)",
            "    at resolveFieldValueOrError (/wiki/node_modules/graphql/execution/execute.js:502:18)",
            "    at resolveField (/wiki/node_modules/graphql/execution/execute.js:460:16)",
            "    at executeFields (/wiki/node_modules/graphql/execution/execute.js:297:18)",
            "    at collectAndExecuteSubfields (/wiki/node_modules/graphql/execution/execute.js:748:10)",
            "    at completeObjectValue (/wiki/node_modules/graphql/execution/execute.js:738:10)",
            "    at completeValue (/wiki/node_modules/graphql/execution/execute.js:626:12)",
            "    at /wiki/node_modules/graphql/execution/execute.js:527:16",
            "    at runMicrotasks (<anonymous>)",
            "    at processTicksAndRejections (node:internal/process/task_queues:96:5)"
          ]
        }
      }
    }
  ],
  "data": {
    "pages": {
      "create": null
    }
  }
}

**Comment by NGPixel** [comment, marked as answer]
Make sure you have a valid token, with the proper rights, that isn't expired.

**Reply by gmouawad** [reply]
thanks NGPixel, I have created a new API token and make sure it has Full Access rights. The page is created now.
I didn't think about it because I thought I have already selected the Full Access checkbox.
Appreciate it.
